exam_2/src/types/TranslateForm.types.ts

```diff
@@ -0,0 +1,25 @@
+export interface ILangsList{
+    langList: String[],
+    handleLangChange?: (e:any) => void
+}
+
+export interface IState{
+    langFrom: string,
+    langTo: string,
+    langList: Array<String>,
+    langNamesList: Array<String>,
```

Comment:
Так, кажется, лаконичнее смотрится

```suggestion
    langNamesList: string[],
```